aws/cloudformation/ami-manager.js

```diff
@@ -0,0 +1,112 @@
+var response = require('cfn-response');
+/** Takes an AWS CloudFormation stack name and instance id and returns the newly-created AMI ID. **/
+exports.handler = function (event, context) {
+  console.log("REQUEST RECEIVED:\n", JSON.stringify(event));
+
+  var stackName = event.ResourceProperties.StackName;
+  var instanceId = event.ResourceProperties.InstanceId;
+  var instanceRegion = event.ResourceProperties.Region;
+
+  var responseStatus = "FAILED";
+  var responseData = {};
+  var physicalId = event.PhysicalResourceId;
+
+  function error(err, msg) {
+    responseData = {Error: msg};
+    console.log(responseData.Error + ":\n", err);
+    response.send(event, context, responseStatus, responseData, physicalId);
+  }
+
+  var AWS = require("aws-sdk");
+  var ec2 = new AWS.EC2({region: instanceRegion});
+
+  console.log("REQUEST TYPE:", event.RequestType);
+  if (event.RequestType == "Delete") {
```

Comment:
It seems the only event.RequestType we handle is Delete., are the others not important?  Could you explicitly check for the other types that we know are safe and thow if there is an unexpected even type?